azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/launcher/CallingCompositeJavaLauncher.java

```diff
@@ -27,7 +28,7 @@ public CallingCompositeJavaLauncher(final Callable<String> tokenRefresher) {
     }
 
     @Override
-    public void launch(final Context context,
+    public void launch(final MainActivity context,
```

Comment:
Should we also rename parameter: final MainActivity mainActivity ?
Same question to CallingCompositeKotlinLauncher.kt and CallingCompositeLauncher.java

Reply:
yeah I can renamed, but I'm not sure if I should do here or the other pr since I renamed, can I follow up on this after they are both merged?

Reply:
nvm, will merge up (see the other is merged already) and update

Reply:
👍